tests/test-users-api.js

```diff
@@ -165,17 +162,29 @@ describe("users api", async() => {
             assert.strictEqual(response.data.State, State.SUCCESS)
             assert.strictEqual(response.data.DeletedUserId, new_user_id)
         })
-        it('10. should return status 500 and State will be error when called incorrectly', async() => {
+        it('10. should return status 500 and State will be error when called with wrong id\'s format', async() => {
             const response = await axios.delete( `${process.env.API_URL}/users/1`).catch(error => {
                 assert.strictEqual(error.response.status, 500)
                 assert.strictEqual(error.response.data.State, State.ERROR)
                 assert.strictEqual(error.response.data.ErrorMessage, ErrorResponseMessage)
             })
-
+        })
+        it('11.should return status 500 and State will be error when called with wrong id\'s value', async() => {
+            let new_id = null
+            if(new_user_id.charAt(35) != '1') {
```

Comment:
There is an easier way to change Id, but can we 100% sure the changed Id hasn't already in the database?

Reply:
I fixed it